packages/react-devtools-shared/src/devtools/views/Components/InspectedElementView.js

```diff
@@ -75,6 +75,7 @@ export default function InspectedElementView({
           <InspectedElementBadges
             hocDisplayNames={element.hocDisplayNames}
             compiledWithForget={element.compiledWithForget}
+            nativeTag={inspectedElement.nativeTag}
```

Comment:
nit: destructure with the rest of inspectedElement?

https://github.com/facebook/react/blob/f134b3993a84d53cc99fe66b426ba13548f142ec/packages/react-devtools-shared/src/devtools/views/Components/InspectedElementView.js#L57-L58